CONTRIBUTING.md

```diff
@@ -6,36 +6,72 @@ For major changes, please open an issue to discuss what you would like to change
 As of writing this document, the project is still in a pre-1.0 state.
 This means that the API is not yet stable and breaking changes may occur frequently.
 
-## Pull Requests
+## Prerequisites
+
+- [`rust`](https://www.rust-lang.org/tools/install)
+- [`docker`](https://docs.docker.com/engine/install/)
+  - You will need this to test the CI/CD pipelines locally or to use Earthly to run various project commands.
+  - You may need to start the docker daemon before running any commands. You can do this with `sudo systemctl start docker`.
+  - You may need to add your user to the `docker` group to run docker commands without `sudo`. You can do this with `sudo usermod -aG docker $USER`.
```

Comment:
This didn't work for me without rebooting my whole computer so I think you should add a sentence telling them to reboot after this.

Reply:
I think you only need to re-login for this to work. Group memberships are checked at login. Rebooting will do this as well.